Fakebook.Posts/Fakebook.Posts.IntegrationTests/ClientActions/PostCreation.cs

```diff
@@ -113,15 +115,38 @@ public async Task PostAsync_InvalidPost_BadRequest()
 
             client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Test");
 
-            NewPostDto newPost = new() { Content = "Valid Content" };
+            // Initialize a string, and then use a forloop to add to the string until it is one longer than the allowed post size
 
-            StringContent stringContent = new(System.Text.Json.JsonSerializer.Serialize(newPost), Encoding.UTF8, "application/json");
+            StringBuilder longPost = new StringBuilder();
+
+            for (int i = 0; i < Constants.PostMaxLength + 1; i++)
+            {
+                longPost.Append("X");
+            }
+
+            //Create New Post DTO objects that will be sent to the API.
+
+            NewPostDto invalidPostTooLong = new() { Content = longPost.ToString() };
+            NewPostDto invalidPostNoContent = new() { Content = "" };
+
+            //  Serializes request object into json format.
+
+            StringContent invalidStringTooMuchContent = new(System.Text.Json.JsonSerializer.Serialize(invalidPostTooLong), Encoding.UTF8, "application/json");
```

Comment:
use using directives instead of full names

Reply:
Fixed it in every place  I could see, I'll do that for the other files too as I go along